mlir/include/mlir/Dialect/Utils/StaticValueUtils.h

```diff
@@ -28,6 +28,10 @@ namespace mlir {
 /// with attribute with value `0`.
 bool isZeroIndex(OpFoldResult v);
 
+/// Return true if `v` is an IntegerAttr with value `1` of a ConstantIndexOp
```

Comment:
Drop the part about `ConstantIndexOp`. This helper extracts integer attributes regardless of the type of operation.